### PR TITLE
chore(samples): add creating managing windows instances samples

### PR DESCRIPTION
**Pull request by FrodoTheTrue**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-compute/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by FrodoTheTrue** [comment]
@savijatv Could you please review the PR?

File: samples/instances/windows/creating-managing-windows-instances/createFirewallRuleForWindowsActivationHost.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        name: firewallRuleName,
+        allowed: [
+          {
+            IPProtocol: 'tcp',
+            ports: ['1688'],
+          },
+        ],
+        direction: 'EGRESS',
+        network: networkName,
+        destinationRanges: ['35.190.247.13/32'],
```
Comment (tmatsuo) [inline samples/instances/windows/creating-managing-windows-instances/createFirewallRuleForWindowsActivationHost.js:50]: Is this IP address guaranteed to work in the future?

Comment (FrodoTheTrue) [reply samples/instances/windows/creating-managing-windows-instances/createFirewallRuleForWindowsActivationHost.js:50]: I believe yes, it's officially documented network: https://cloud.google.com/compute/docs/instances/windows/creating-managing-windows-instances#kms-server for kms.windows.googlecloud.com

File: samples/test/creatingManagingWindows.test.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const deleteFirewallRule = async (projectId, firewallRule) => {
+  const [response] = await firewallsClient.delete({
+    project: projectId,
+    firewall: firewallRule,
+  });
+  let operation = response.latestResponse;
+
+  // Wait for the create operation to complete.
```
Comment (tmatsuo) [inline samples/test/creatingManagingWindows.test.js:39]: The comment says "create" but the operation is "delete"?

File: samples/test/creatingManagingWindows.test.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+const deleteRoute = async (projectId, routeName) => {
+  const [response] = await routesClient.delete({
+    project: projectId,
+    route: routeName,
+  });
+  let operation = response.latestResponse;
+  const operationsClient = new compute.GlobalOperationsClient();
+
+  // Wait for the create operation to complete.
```
Comment (tmatsuo) [inline samples/test/creatingManagingWindows.test.js:56]: ditto

File: samples/test/creatingManagingWindows.test.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  const instanceName = generateTestId();
+  const firewallRuleName = generateTestId();
+  const networkName = 'global/networks/default-compute';
+  const subnetworkName = 'regions/europe-central2/subnetworks/default-compute';
+  const routeName = generateTestId();
+  const zone = 'europe-central2-b';
+  const machineType = 'n1-standard-1';
+  const sourceImageFamily = 'windows-2012-r2';
+
+  after(async () => {
```
Comment (tmatsuo) [inline samples/test/creatingManagingWindows.test.js:75]: Not a big deal, but can we do this *before* instead of after? In some edge cases, it will prevent quota errors.

**Review by tmatsuo — APPROVED** [review]
